src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +14,11 @@
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        return users.stream()
+                .map(Optional::of)
+                .filter(user -> user.get().matchName(name))
+                .findFirst()
+                .orElse(Optional.of(DEFAULT_USER))
+                .get();
```

Comment:
```suggestion
        return users.stream()
                .filter(user -> user.matchName(name))
                .findFirst()
                .orElseGet(() -> DEFAULT_USER);
```
findFirst()의 반환값이 Optional이기 때문에 Optional 변환을 명시적으로 하지 않아도 될 것 같습니다 😊